src/aksharamukha/index.ts

```diff
@@ -1,194 +1,188 @@
-		let pyodide = opts?.pyodide;
-		if (pyodide == null) {
-			if (this._isTestEnv) {
-				pyodide = await loadTestPyodide();
-			} else {
-				pyodide = await loadPyodide();
-			}
-		}
-
-		await pyodide.loadPackage('micropip');
-		const micropip = pyodide.pyimport('micropip');
-
-		if (isNode) {
-			const fs = await import('fs');
-
-			for (const wheel of wheels) {
-				let wheelData: Buffer<ArrayBuffer>;
-				const currentDir = getCurrentDir()
-
-				try {
-					const wheelPath = `${currentDir}/${wheel}`;
-					wheelData = fs.readFileSync(wheelPath);
-				} catch (e) {
-					console.warn(`Wheel file missing in script directory, trying ../../downloads: ${e}`);
-					const wheelPath = `${currentDir}/../../downloads/${wheel}`;
-					wheelData = fs.readFileSync(wheelPath);
-				}
-
-				pyodide.FS.writeFile(`/tmp/${wheel}`, wheelData);
-			}
-
-			await micropip.install(wheels.map(wheel => `emfs:/tmp/${wheel}`), { keep_going: true });
-
-			for (const wheel of wheels) {
-				pyodide.FS.unlink(`/tmp/${wheel}`);
-			}
-		} else {
-			try {
-				const scriptPath = this._currentScript.src;
-				const parentPath = scriptPath.substring(0, scriptPath.lastIndexOf('/'));
-				await micropip.install(wheels.map(wheel => `${parentPath}/${wheel}`), { keep_going: true })
-			} catch {
-				await micropip.install(wheels.map(wheel => `${wheelBaseURL}/${wheel}`), { keep_going: true })
-			}
-		}
-
-		pyodide.runPython(`from aksharamukha import *`); // Pre-import to speed up further calls
-		return new Aksharamukha(pyodide);
-	}
-
-	public async test() {
-		const result = await this.pyodide.runPythonAsync(`1 + 1`);
-		if (result !== 2) {
-			throw new Error('Pyodide not functioning correctly.');
-		}
-	}
-
-	public process(
-		src: string,
-		tgt: string,
-		txt: string,
-		{
-			nativize,
-			param,
-			preOptions,
-			postOptions
-		}: ProcessProps = defaultProcessProps
-	) {
-		const cmd = buildCMD({
-			src,
-			tgt,
-			txt,
-			props: {
-				nativize: nativize ?? defaultProcessProps.nativize,
-				param: param ?? defaultProcessProps.param,
-				preOptions: preOptions ?? defaultProcessProps.preOptions,
-				postOptions: postOptions ?? defaultProcessProps.postOptions
-			}
-		});
-		return this.pyodide.runPython(cmd);
-	}
-
-	public async processAsync(
-		src: string,
-		tgt: string,
-		txt: string,
-		{
-			nativize,
-			param,
-			preOptions,
-			postOptions
-		}: ProcessProps = defaultProcessProps
-	) {
-		const cmd = buildCMD({
-			src,
-			tgt,
-			txt,
-			props: {
-				nativize: nativize ?? defaultProcessProps.nativize,
-				param: param ?? defaultProcessProps.param,
-				preOptions: preOptions ?? defaultProcessProps.preOptions,
-				postOptions: postOptions ?? defaultProcessProps.postOptions
-			}
-		});
-		return await this.pyodide.runPythonAsync(cmd);
-	}
+  private pyodide: PyodideInterface;
+  private static _isTestEnv = false;
+  private static _currentScript: HTMLScriptElement | null = null;
+  private static _packagesInstalled = false;
```

Comment:
I think this needs to be removed, now that we don't have a single class-level constructor.